src/assets/thirdPartyServices/mozi/gene_annotation_service/result/index.js

```diff
@@ -121,7 +121,7 @@ const AnnotationResult = props => {
       </div>
       {isVisualizerShown && (
         <Visualizer
-          graph={{ ...response.result.elements }}
+          graph={JSON.parse(JSON.stringify({ ...response.result.elements }))}
```

Comment:
why do you stringify the JSON and parse the same immediately. is there any significance for it?

Reply:
It is to make a deep clone of the object. Otherwise, only a reference will be passed and the original object will be modified.

Reply:
:+1: thanks for the info..